templates/patterns/timeline/pattern-timeline.html.twig

```diff
@@ -26,6 +26,11 @@
     {{ title }}
   </h2>
 {% endif %}
+{% if introduction %}
+  <div id="timeline-introduction">
```

Comment:
I think we should apply text related ecl classes because now the text is black.

Reply:
That's how we render that particular paragraph field everywhere.

Reply:
```suggestion
  <div class="timeline-introduction">
```
"id" attribute has to be unique on the page. Since it is a paragraph, then few elements with the same id can be presented on the page

Reply:
It's not needed. Please check how we render this paragraph field.

Reply:
Once we apply the text formats, the corresponding wrapper with the ecl class will be applied on render.